core-kotlin/src/main/java/com/amplifyframework/kotlin/auth/Auth.kt

```diff
@@ -86,11 +87,16 @@ interface Auth {
      * be used to send them a new one.
      * @param username A login identifier e.g. `tony44`; or an email/phone number,
      *                 depending on configuration
+     * @param options Advanced options such as a map of auth information for custom auth,
+     *                If not provided, default options will be used
      * @return A sign-up result; if the code is requested, typically the result will
      *         include a next step requiring confirmation of the re-sent code.
      */
     @Throws(AuthException::class)
-    suspend fun resendSignUpCode(username: String): AuthSignUpResult
+    suspend fun resendSignUpCode(
+        username: String,
+        options: AuthResendSignUpCodeOptions = AuthResendSignUpCodeOptions.defaults()
+    ): AuthSignUpResult
```

Comment:
Same concern as the above - can you think of a way to introduce this in a backward compatible way?

Reply:
The newly added parameter has a default option. This is backwards compatible.